cobbler/cobblerd.py

```diff
@@ -82,10 +82,18 @@ def do_xmlrpc_rw(cobbler_api: CobblerAPI, port):
     server.logRequests = 0      # don't print stuff
     logger.debug("XMLRPC running on %s", port)
     server.register_instance(xinterface)
+    start_time = ""
+    try:
+        import psutil
+        p = psutil.Process(os.getpid())
+        start_time = " in %s seconds" % str(time.time() - p.create_time())
+    except ModuleNotFoundError:
+        # This is not critical, but debug only - just install python3-psutil
+        pass
 
     while True:
         try:
-            logger.info("Cobbler startup complete")
+            logger.info("Cobbler startup completed" + start_time)
```

Comment:
The reason that I am so strict is that Python does not do str replacments when the loglevel is higher then the one in the message. Thus saving us a lot of time at runtime.

```suggestion
            logger.info("Cobbler startup completed %s", start_time)
```

Reply:
Argh. Yep, I am stumbling over this again and again. This was not on purpose, I cherry-picked my first try and forgot to adopt it. Good catch, thanks.